sdk/src/main/java/com/hedera/hashgraph/sdk/FeeData.java

```diff
@@ -0,0 +1,99 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import javax.annotation.Nullable;
+
+public class FeeData {
+
+    @Nullable
+    private FeeComponents nodeData;
+    @Nullable
+    private FeeComponents networkData;
+    @Nullable
+    private FeeComponents serviceData;
+
+    public FeeData() {
+        nodeData = null;
+        networkData = null;
+        serviceData = null;
+    }
+
+
+
+
+
+    // getters
+
+    boolean hasNodeData() {
```

Comment:
No need to have `has*` methods; it's not something we support in the rest of the SDK.